src/OmniSharp.Roslyn/WorkspaceServices/OmniSharpSymbolRenamedCodeActionOperationFactoryWorkspaceServiceFactory.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Composition;
+using System.Reflection;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace OmniSharp
+{
+    [Shared]
+    [ExportWorkspaceServiceFactoryWithAssemblyQualifiedName("Microsoft.CodeAnalysis.Features", "Microsoft.CodeAnalysis.CodeActions.WorkspaceServices.ISymbolRenamedCodeActionOperationFactoryWorkspaceService")]
+    public class OmniSharpSymbolRenamedCodeActionOperationFactoryWorkspaceServiceFactory : IWorkspaceServiceFactory
```

Comment:
Just to explain what is going on here - we need to do this because `NamingStyleCodeFixProvider ` internally tries to resolve `ISymbolRenamedCodeActionOperationFactoryWorkspaceService`. That doesn't ship together with Roslyn but instead is only implemented inside `Microsoft.VisualStudio.LanguageServices`, which we can't really reference (it has the whole tree of VS specific GUI dependencies). 

On the other hand, `ISymbolRenamedCodeActionOperationFactoryWorkspaceService` is internal, so implementing is by hand requires a bit twisting